transport-classes-io_uring/src/main/java/io/netty/channel/uring/IoUringIoHandler.java

```diff
@@ -134,21 +128,29 @@ public int run(IoExecutorContext context) {
         } else {
             submissionQueue.submit();
         }
-        // we might call submitAndRunNow() while processing stuff in the completionArray we need to
-        // add the processed completions to processedPerRun as this might also be updated by submitAndRunNow()
-        processedPerRun += drainAndProcessAll(completionQueue, this::handle);
-
-        // Let's submit one more time as the completions might have added things to the submission queue.
-        submissionQueue.submit();
+        for (;;) {
```

Comment:
@chrisvest @franz1981 This change is important and fixes the 10 - 15 % perf drop I was seeing and why I did https://github.com/netty/netty/pull/14711 in the first place.